pincer/objects/message/file.py

```diff
@@ -106,7 +166,7 @@ def from_pillow_image(
             )
 
         if image_format is None:
-            image_format = os.path.splitext(filename)[1][1:]
+            image_format = _get_file_extension(filename)
 
             if image_format == "jpg":
                 image_format = "jpeg"
```

Comment:
Shouldn't the 2nd if statement be unindented?

Reply:
good call